app/src/main/java/com/nmakademija/nmaakademija/entity/Article.java

```diff
@@ -28,4 +29,41 @@ public String getTitleImage() {
     public String getContent() {
         return content;
     }
+
+    @Override
+    public int describeContents() {
```

Comment:
These methods has been autogenerated with https://github.com/mcharmas/android-parcelable-intellij-plugin

There is many projects which can automatically generate these methods on compile time, but I didn't wanted to introduce new library.